uploader.json

```diff
@@ -304,6 +304,11 @@
             ],
             "symbolic_name": "heat_crontab"
         },
+        {
+            "command": "/bin/hostname",
```

Comment:
@xiangce this command should match the command in insights-core `/usr/bin/hostname`.

Reply:
@bfahr - Thanks for that. But it should be `/bin/hostname` in case the `/usr/bin/hostname` is not available. I'll fix the one in `insights-core` instead of changing the specs in uploader.json.

Reply:
@bfahr - I open https://github.com/RedHatInsights/insights-core/issues/2175 for that.

Reply:
The https://github.com/RedHatInsights/insights-core/issues/2175 is merged, Could you please to check this PR again?